server/routes/alpha/activity-list/handlers/selectActivityLocationRouteHandler.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from 'express'
+import { DateTime } from 'luxon'
+import { switchDateFormat, getCurrentPeriod } from '../../../../utils/utils'
+import PrisonService from '../../../../services/prisonService'
+
+export default class SelectActivityLocationRouteHandler {
+  constructor(private readonly prisonService: PrisonService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { user } = res.locals
+    const { prisonId } = req.query as Record<string, string>
+    const nowDt = DateTime.now()
+    const bookedOnDay = req.query?.date || nowDt.toLocaleString(DateTime.DATE_SHORT) // moment().format('DD/MM/YYYY')
```

Comment:
I think there was a slight preference for `date-fns` as the replacement for `moment` - but `luxon` was a close second choice.